buildscripts/migrate/Makefile.mk

```diff
@@ -0,0 +1,32 @@
+
+# Specify the name for the binaries
+MIGRATE=migrate
```

Comment:
name seems to be very generic.. does the same image expected to do vol migration as well?

Reply:
Yes the image is `pool` as subcommand for now. We can add `cstor-volume` and `jiva-volume` subcommands for migrating csi volumes.